docs/building.md

```diff
@@ -28,25 +28,22 @@ For Linux a few additional packages beyond the usual build essentials are needed
 
 ### Windows
 
-- Make sure the MSVC Build Tools are installed: `winget install Microsoft.VisualStudio.2019.BuildTools`
+- Make sure the MSVC Build Tools are installed: `winget install Microsoft.VisualStudio.2022.BuildTools`
 - (optional) make sure Qt is installed and `qmake` is in the `Path`
 
-### NodeJS API (optional)
-
-To use Slint from Node.JS, the following extra dependencies are needed.
-
-- **node** (including npm) At this time you will need to use the LTS version.
-- **python**
-
-Node.JS support is not built by default! Check below for the extra commands
-to run.
-
 ### C++ API (optional)
 
 To use Slint from C++, the following extra dependencies are needed:
 
 - **[cmake](https://cmake.org/download/)** (3.19 or newer)
-- A C++ compiler that can do C++20 (e.g., **MSVC 2019 16.6** on Windows)
+- A C++ compiler that can supports C++20 (e.g., **MSVC 2019 16.6** on Windows)
```

Comment:
"that supports", not "that can supports"

You said MSVC2022 above, do you need to adapt this here as well?

Reply:
Hm, I didn't change it because that's only a requirement when using Skia. But I might as well bump it...